### ISSUE TITLE
Subliminal won't search subtitles

### ISSUE THREAD

**Issue by Tarok9** [Diaoul/subliminal#949]
Hey,

When I search in subliminal over terminal i get this error:
Usage: subliminal [OPTIONS] COMMAND [ARGS]...
Try "subliminal --help" for help.

Error: no such option: -1

I tried reinstalling subliminal, it didn't help.
How can I fix this?

Thank you in advance.

**Comment by ratoaq2** [comment]
More information is needed in order to help you.

What's your operation system? how did you install it? How are you reinstalling it?

**Comment by Tarok9** [comment]
Hey,

I use mint 19 Tara, I installed subliminal from a terminal, I tried
reinstalling it and installing it conventional way but it still didn't
work.
If you need any more informations, I'll provide it.

Kind regards.

**Comment by ratoaq2** [comment]
Which command you used to install subliminal? What's the python version, pip version and the subliminal version?

**Comment by Tarok9** [comment]
$ [sudo] pip install subliminal


I installed pip first than tried with command above, but with no success
I also tried installing with the pip3 command but it didn't work either.
My pyhton version is 3.
Subliminal version is 0.7.6

**Comment by ratoaq2** [comment]
That's the issue, subliminal 0.7.6 is very very old.
Uninstall it and install the one using pip

You should get subliminal 2.0.5 or latest develop 2.1.0

**Comment by Tarok9** [comment]
I deleted and reinstalled subliminal from pip command.
When I downloaded  a subtitle for the first movie, it downloaded it but the
subtitle didn't show in the media player (I use mpv).
I tried downloading for another movie but it gets this message out when I
try to download the subtitle:
"Usage: subliminal download [OPTIONS] PATH...
Try "subliminal download --help" for help.

Error: no such option: -1"

Also I checked my Python version it's 2.7.


pet, 22. velj 2019. u 12:45 Rato <notifications@github.com> napisao je:
